index.html

```diff
@@ -273,5 +273,6 @@ <h1><span class="name"></span> Error</h1>
                   arrow_2_circlepath</i></button>
     </nav>
 </template>
+<script type="text/javascript" src="/cacheBuster.ts"></script>
```

Comment:
I'm afraid this won't work. Any javascript will run AFTER the browser already cached index.html and therefore will not validate the cache.
I believe the only solution is to use the cache meta tags.

Reply:
i Tired and the app wont load for some reason so i tried to bypass it , i will try again .